python/dask_cudf/dask_cudf/tests/test_sort.py

```diff
@@ -49,3 +49,19 @@ def test_sort_repartition():
     new_ddf = ddf.shuffle(on="a", ignore_index=True, npartitions=3)
 
     dd.assert_eq(len(new_ddf), len(ddf))
+
+
+@pytest.mark.parametrize("by", ["a", "b", ["a", "b"]])
+def test_sort_values_with_nulls(by):
+    df = cudf.DataFrame(
+        {
+            "a": list(range(50)) + [None] * 50 + list(range(50, 100)),
+            "b": [None] * 100 + list(range(100, 150)),
+        }
+    )
```

Comment:
Would it be possible to add an assertion that Nulls are in the divisions in this test ?

Reply:
Yeah, I'll add that in now